### DISCUSSION TITLE
两岸四地并列之中立称法

### DISCUSSION THREAD

**Discussion by liyishuai** [sparanoid/chinese-copywriting-guidelines#184] [Ideas]
## 背景
#117 引发了关于如何并列称呼两岸四地的争论。鉴于此并列关系预计还会存在相当一段时间（起码四地的货币都不一致），试讨论之：
- `中国大陆、中国香港、中国澳门、中国台湾`首先在China (People's Republic of)官方语境下就是不规范的用法，因为`大陆`只能与`台湾`并列，而与港澳并列时应称`内地`；
- `中国台湾`、`Taiwan province of China`用法在China (Republic of) (a.k.a. Taiwan)的接受度存疑，且忽略了福建省金门县和连江县；
- `中国、香港、澳门、台湾`在内地不太政治正确，不正确程度取决于京港关系；
- `中国、台湾`在大陆极其政治不正确。
## 提案
| 中文 | 英文 |
|---|---|
|中国内地|China mainland|
|香港特别行政区|Hong Kong SAR|
|澳门特别行政区|Macau SAR|
|台湾|Taiwan|

其中`mainland`用小写，以表明“内地是中国的真子集”，而非名为“Mainland China”的概念；同时保证开头为C，防止字母序紊乱。
香港和澳门加注`特别行政区`，以表明与China (People's Republic of)在法理上公认的管辖关系。
`台湾`与`香港`、`澳门`并列，在大中华区和国际上的接受度尚可。为此，避免使用`中国香港`、`Macau, SAR China`等提及“中国”概念的表达方式，在一定程度上防止“只有‘台湾’前面没加‘中国’”之找茬。

---
后记：愿世界和平、人类有朝一日互相理解。

**Comment by lance111l** [comment]
IN UN , ONLY  People's Republic of China    no Taiwan     and always no Taiwan
If  small Taiwan tried  to be independent   , it will be heavily bombed  and no more chance in the world

**Comment by HumJ0218** [comment]
这个问题不是这里有能力讨论的，排版方案不该涉及对内容本身的讨论，更何况在现实背景下，文案中应避免出现此类并列。

至于 #117 中的情形，我的建议是以不必参照任何建议，以网站实际显示为准。

**Reply by liyishuai** [reply]
我的部分意见与您相左：
- > 这个问题不是这里有能力讨论的

  讨论不需要“能力”。GitHub是简体中文互联网为数不多可以自由讨论的“场所”。
- > 排版方案不该涉及对内容本身的讨论

  本项目涵盖了文案和排版意见，故内容的呈现形式不属于离题。
- > 更何况在现实背景下，文案中应避免出现此类并列。

  在现实背景下，此种并列是不可避免的，例如《[习近平：在二〇二三年春节团拜会上的讲话](http://www.gov.cn/xinwen/2023-01/20/content_5738258.htm)》：
  > 首先，我代表党中央和国务院，向大家致以节日的美好祝福！向全国各族人民，向**香港特别行政区同胞、澳门特别行政区同胞、台湾同胞和海外侨胞**拜年！祝大家新春愉快！


> 至于 #117 中的情形，我的建议是以不必参照任何建议，以网站实际显示为准。

赞同。内容的呈现形式属言论自由，我开贴只是为了讨论如何呈现可以尽量避免冒犯到受众群体。
